src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerTopPanel.xaml

```diff
@@ -246,7 +244,7 @@
           Unchecked="_checkboxVulnerabilities_Unchecked"
           Visibility="Hidden"
           IsChecked="False">
-          <TextBlock Text="{x:Static nuget:Resources.Checkbox_Show_Vulnerable_Only}" />
+          <TextBlock TextTrimming="CharacterEllipsis" Text="{x:Static nuget:Resources.Checkbox_Show_Vulnerable_Only}" />
```

Comment:
Thanks for working to improve the reflow layout issues in PM UI. I believe you could split this into a separate PR and get it merged and separated from the README work.